Bottom: 546230af519e3539f1e7f13a89e7b1d050a64a76
Top:    2eee8982fcd1e14263d58ade8080438507ed1862
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 16:56:59 +0000

Refresh of find-c-use-setthing-rather-tha

---

diff --git a/src/find.c b/src/find.c
index d10499d..e63572e 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3480,10 +3480,8 @@ drc_callback (DataType *data, LayerType *layer, PolygonType *polygon,
   PinType *pin = (PinType *) ptr2;
   PadType *pad = (PadType *) ptr2;
 
-  thing_type = type;
-  thing_ptr1 = ptr1;
-  thing_ptr2 = ptr2;
-  thing_ptr3 = ptr2;
+  SetThing (type, ptr1, ptr2, ptr2);
+
   switch (type)
     {
     case LINE_TYPE:
